hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -20,8 +20,7 @@
 from health_ml.utils.common_utils import (CHECKPOINT_FOLDER,
                                           create_unique_timestamp_id,
                                           DEFAULT_AML_UPLOAD_DIR,
-                                          DEFAULT_LOGS_DIR_NAME,
-                                          parse_model_id_and_version)
```

Comment:
I think this file is the only place we were calling this function? If so, can we delete it now?